README.md

```diff
@@ -18,6 +18,7 @@ You will need the following:
    - Windows and macOS builds can be downloaded directly from [releases](https://github.com/CorsixTH/CorsixTH/releases).
    - Linux and BSD repositories use either corsixth or corsix-th names [packaged versions](https://repology.org/metapackage/corsixth).
    - A Flatpak for Linux users is available on [Flathub](https://flathub.org/apps/details/com.corsixth.corsixth).
+   - A Snap for Linux users is available on [Snapcraft](https://snapcraft.io/corsixth).
```

Comment:
```suggestion
   - A Snap for Linux users is available on [Snapcraft](https://snapcraft.io/corsixth) [(support page)](https://github.com/snapcrafters/corsixth).
```

Maybe we could do this as @Alberth289346 suggests @anthonydillon .

Reply:
Trying to have a bit more distinction, please have a look at the 2 commits in https://github.com/Alberth289346/CorsixTH/commits/split-suppliers/

Reply:
I have committed the suggestion but agree it would be more clear to provide the support link for all package managers, as the linked commit does.

Reply:
> Trying to have a bit more distinction, please have a look at the 2 commits in https://github.com/Alberth289346/CorsixTH/commits/split-suppliers/

If you wanted to, we could open a pr new for your styling commits now :)